ReactAndroid/src/main/java/com/facebook/react/uimanager/UIImplementation.java

```diff
@@ -226,7 +226,8 @@ public void updateView(int tag, String className, ReadableMap props) {
    */
   public void synchronouslyUpdateViewOnUIThread(int tag, ReactStylesDiffMap props) {
     UiThreadUtil.assertOnUiThread();
-    mOperationsQueue.getNativeViewHierarchyManager().updateProperties(tag, props);
+    mOperationsQueue.enqueueNonBatchedUpdateProperties(tag, props);
+    mOperationsQueue.dispatchNonBatchedViewUpdates();
```

Comment:
This is needed since we can't rely on the flush that happens in the frame callback because it will be a frame behind, this is especially visible with animations that use an `Animated.event`